packages/js/src/settings/routes/site-representation.js

```diff
@@ -44,6 +45,11 @@ const SiteRepresentation = () => {
 	const mastodonPremiumLink = useSelectSettings( "selectLink", [], "https://yoa.st/get-mastodon-integration" );
 	const mastodonUrlLink = useSelectSettings( "selectLink", [], "https://yoa.st/site-representation-mastodon" );
 
+	const handleAddProfile = useCallback( async( arrayHelpers ) => {
+		await arrayHelpers.push( null, "" );
```

Comment:
This pushes two values on the array. With the bind you copied, the `null` was the scope you pass to bind (i.e. the value of `this`).
If you press the add profile button, you get a `null` is not allowed warning.